### ISSUE TITLE
Padavan archer-ax20 ?

### ISSUE THREAD

**Issue by finBitorg** [hanwckf/rt-n56u#439]
https://www.tp-link.com/us/home-networking/wifi-router/archer-ax20/

**Comment by finBitorg** [comment]
https://www.mediatek.com/blog/wi-fi-6-home-routers-powered-by-mediatek

**Comment by paldier** [comment]
It's bcm6755

**Comment by gbtota** [comment]
Why is it on Mediatek website then? 

"The first wave of Wi-Fi 6 home routers to market, powered by MediaTek:

    D-Link AXO AX1800 (DIR-X1860)
    TP-Link Archer AX20 (AX1800 Dual-Band Wi-Fi 6 Router)
    Buffalo AX1800 (WSR-1800AX4) - in Japan"

**Comment by paldier** [comment]
maybe v1 is bcm and v1.2 is mtk.
7621+7915

**Comment by gbtota** [comment]
Yes most likely; here it specifically states that AX1800: 7621+7915D

https://www.mediatek.com/products/homeNetworking/mt7621

**Comment by paldier** [comment]
The v1 is not sold in mainland China, it has a better cpu, you must go China to buy v1.2(possibly not, and no GPL code).
